src/components/ItemCard/item-card.css

```diff
@@ -0,0 +1,150 @@
+.css-1elwnq4-MuiPaper-root-MuiAccordion-root {
+  border: 1px solid #e6e7e7;
+  border-radius: 0px 14px 14px 0px !important;
+  border-left: 5px solid;
+  padding: 15px;
+  margin-bottom: 25px;
+}
+
+.css-1elwnq4-MuiPaper-root-MuiAccordion-root,
+.css-sh22l5-MuiButtonBase-root-MuiAccordionSummary-root {
+  box-shadow: none !important;
+  cursor: default !important;
+}
+
+.css-1elwnq4-MuiPaper-root-MuiAccordion-root:before {
+  display: none;
+}
+
+.css-o4b71y-MuiAccordionSummary-content {
```

Comment:
This styling doesn't seem to be showing up on the build preview:

![stylingissue-onbuild](https://user-images.githubusercontent.com/82783679/156219464-50f52728-4a1f-420c-b24e-7d3e28b45a26.jpg)

But everything looks as it should in the dev environment, so I think it may come down to those hashes in the class name - I believe they're unstable between builds. [Here's the reference from MUI.](https://mui.com/customization/how-to-customize/#overriding-nested-component-styles) Should still work if you nerf out the `.css-[hash]-` prefixes.

Reply:
Such a great catch, Rafa, thanks!! Just committed the fixes, thanks!